schema/alpha/artifacts/artifact.json

```diff
@@ -1,6 +1,23 @@
 {
     "$id": "http://github.com/kubeflow/metadata/schema/alpha/artifacts/artifact.json",
     "$schema": "http://json-schema.org/draft-07/schema#",
+    "examples": [{
+        "category": "artifact",
+        "id": "123",
+        "kind": "model",
```

Comment:
Is this a free flowing list of key-value pairs? Not checked against a specific schema e.g. I can define my own "kind"?

Reply:
> Is this a free flowing list of key-value pairs? 

It is right now. But you can constrain it to only known properties by setting`  "additionalProperties": false` in this schema.
http://json-schema.org/understanding-json-schema/reference/object.html#properties

> Not checked against a specific schema e.g. I can define my own "kind"?

The properties of this example is checked against the `allOf` section in this file. For any known properties in the `allOf`, it has to satisfy all constraints.

You can define you own `model`. e.g. see `schema/alpha/artifacts/model.json` file, which defines  `model` artifact for `kubeflow.org`. You can add a schema for your own model type.